openassessment/fileupload/backends/base.py

```diff
@@ -0,0 +1,130 @@
+import abc
+
+from django.conf import settings
+
+from ..exceptions import FileUploadInternalError
+from ..exceptions import FileUploadRequestError
+
+
+class Settings(object):
+    """Store settings related to file upload
+
+    The following settings are used:
+
+        FILE_UPLOAD_STORAGE_BUCKET_NAME (str, required): name of the bucket
+        (AWS or local file directory) to which content will be uploaded.
+
+        FILE_STORAGE_STORAGE_PREFIX (str, defaults to "submissions_attachment"): this
+        will be used to prefix all stored file names. The specified file
+        prefix for the storage must be publicly viewable or all uploaded files
+        will not be seen.
+    """
+
+    @classmethod
+    def get_bucket_name(cls):
+        bucket_name = getattr(settings, "FILE_UPLOAD_STORAGE_BUCKET_NAME", None)
+        if not bucket_name:
+            raise FileUploadInternalError("No bucket name configured for FileUpload Service.")
+        return bucket_name
+
+    @classmethod
+    def get_prefix(cls):
+        """Return the prefix for stored files.
+
+        Defaults to 'submissions_attachments'.
+        """
+        default_file_upload_storage_prefix = "submissions_attachments"
```

Comment:
Sorry - I wasn't clear. I meant consider moving this default up into `Settings` class scope as `DEFAULT_FILE_UPLOAD_STORAGE_PREFIX`. Then remove the references to `submissions_attachments` in the docstrings above. That way, other users can more easily change the default storage prefix and the tests can also use this default variable in `Settings`.